File: openapi-generator/dart-jaguar/lib/model/door_response.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:openapi/model/door_response_sensors.dart';
part 'door_response.jser.dart';

class DoorResponse {
  
  @Alias('groupId')
  final int groupId;
  
  @Alias('sensors')
  final List<DoorResponseSensors> sensors;
  

  DoorResponse(
      

{
     this.groupId = null,  
     this.sensors = const [] 
    
    }
  );

  @override
  String toString() {
    return 'DoorResponse[groupId=$groupId, sensors=$sensors, ]';
  }
}

@GenSerializer(nullableFields: true)
class DoorResponseSerializer extends Serializer<DoorResponse> with _$DoorResponseSerializer {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/DoorResponse.scala

package org.openapitools.client.model


case class DoorResponse (
    _groupId: Option[Long],
    _sensors: Option[List[DoorResponseSensors]]
)
object DoorResponse {
    def toStringBody(var_groupId: Object, var_sensors: Object) =
        s"""
        | {
        | "groupId":$var_groupId,"sensors":$var_sensors
        | }
        """.stripMargin
}
